tests/test_formula.py

```diff
@@ -73,3 +73,14 @@ def test_invalid_formula(self):
     def test_invalid_materializer(self, formula_expr, data):
         with pytest.raises(FormulaMaterializerInvalidError):
             formula_expr.get_model_matrix(data, materializer=object())
+
+
+    def test_bs(self):
```

Comment:
I threw this end-to-end test into the `test_formula.py` because I didn't know a better place.

Reply:
The general approach I've taken to testing when it comes to transforms is to trust that the lower-level unit tests have caught anything necessary. At this level I think I'd want to just make sure that the functions are being passed through successfully; which wouldn't be a `bs` specific test.